specs/SensitivityLabel.md

```diff
@@ -0,0 +1,541 @@
+      [out, retval] COREWEBVIEW2_SENSITIVITY_LABEL_KIND* value);
+}
+
+/// Interface for Microsoft Information Protection (MIP) sensitivity labels.
+/// This interface provides specific information about MIP labels, including
+/// label identification and organizational context.
+[uuid(1a562888-3031-5375-b8c5-8afd573e79c8), object, pointer_default(unique)]
+interface ICoreWebView2MipSensitivityLabel : IUnknown {
+  /// The unique identifier for the Microsoft Information Protection label.
+  /// This string contains a GUID that uniquely identifies the specific
+  /// sensitivity label within the organization's MIP policy configuration.
+  /// The GUID is of type GUIDv4 and follows the format:
+  /// `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`
+  /// eg: `3fa85f64-5717-4562-b3fc-2c963f66afa6`
+  ///
+  /// The caller must free the returned string with `CoTaskMemFree`.  See
+  /// [API Conventions](/microsoft-edge/webview2/concepts/win32-api-conventions#strings).
+  [propget] HRESULT LabelId([out, retval] LPWSTR* value);
+
+  /// The unique identifier for the organization that owns the MIP label.
+  /// This string contains a GUID that identifies the Azure Active Directory
+  /// tenant or organization that configured and deployed the sensitivity label.
+  /// The GUID is of type GUIDv4 and follows the format:
+  /// `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`
+  /// eg: `44567f64-8712-1789-ac3f-15aa3f66ab12`
+  ///
+  /// The caller must free the returned string with `CoTaskMemFree`.  See
+  /// [API Conventions](/microsoft-edge/webview2/concepts/win32-api-conventions#strings).
+  [propget] HRESULT OrganizationId([out, retval] LPWSTR* value);
+}
+
+
+/// A collection of ICoreWebView2SensitivityLabel.
+[uuid(2cb85219-0878-5f38-b7e9-769fab6ff887), object, pointer_default(unique)]
+interface ICoreWebView2SensitivityLabelCollectionView : IUnknown {
+  /// The number of elements contained in the collection.
+  [propget] HRESULT Count([out, retval] UINT32* value);
+
+  /// Gets the element at the given index.
+  HRESULT GetValueAtIndex(
+      [in] UINT32 index, 
+      [out, retval] ICoreWebView2SensitivityLabel** value);
+}
+
+/// Event arguments for the `SensitivityLabelChanged` event.
+/// This interface provides information about sensitivity label changes
+/// that occur when web page is loaded or updated in the WebView2 control.
+/// The event args contain the current state of sensitivity label detection
+/// and a collection of all sensitivity labels that have been reported by
+/// the web page via 
+/// [`Page Interaction Restriction Manager`](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/PageInteractionRestrictionManager/explainer.md). 
+[uuid(36de2060-e013-5b03-939b-117d08d0abd5), object, pointer_default(unique)]
+interface ICoreWebView2SensitivityLabelEventArgs : IUnknown {
+  /// Gets a read-only collection of all sensitivity labels detected in the
+  /// current web document. This collection contains instances of sensitivity
```

Comment:
SensitivityLabels is guaranteed to be empty when SensitivityState is None or Undetermined, correct?